### ISSUE TITLE
tmux 3.2: Multi-pane window cancels selection when moving by tmux-easy-motion

### ISSUE THREAD

**Issue by butwerenotthereyet** [IngoMeyer441/tmux-easy-motion#15]
In tmux 3.2, if a window has multiple panes, when you begin a selection and then move via tmux-easy-motion, your selection is cancelled, preventing you from selecting a region via tmux-easy-motion.

These screenshots were taken in a tmux 3.2 client running in Alacritty on macOS.  It was launched like this:

```
$ cat /tmp/tmux.easymotion.conf
set -g @easy-motion-copy-mode-prefix-enabled "1"
set -g @easy-motion-copy-mode-prefix "s"
set -g @easy-motion-prefix-enabled "0"
set -g @easy-motion-dim-style "fg=colour242"
set -g @easy-motion-highlight-style "fg=colour196,bold"
set -g @easy-motion-highlight-2-first-style "fg=#ffb400,bold"
set -g @easy-motion-highlight-2-second-style "fg=#b98300,bold"
run-shell "/tmp/tmux-easy-motion/easy_motion.tmux"
$ tmux -V
tmux 3.2
$ tmux -L test -f /tmp/tmux.easymotion.conf
```

Starting from a session with a single window split vertically where the left pane has `man tmux` running, enter copy mode, jump to a character, begin selection, move a little to see that the selection is active, then use tmux-easy-motion, and jump to a different character.  At this point, you will see that the selection has been cancelled.

![1](https://user-images.githubusercontent.com/58348703/131070853-5c8657cf-0ff4-4d41-8af3-da4f5c8d3db3.png)
![2](https://user-images.githubusercontent.com/58348703/131070855-4c938a13-f915-43e8-9dd3-08f15e92ffd6.png)
![3](https://user-images.githubusercontent.com/58348703/131070856-54b06040-7075-42b9-a191-a9b03fc14993.png)
![4](https://user-images.githubusercontent.com/58348703/131070858-b4fa8e34-16e9-421a-898b-9082df127e44.png)
![5](https://user-images.githubusercontent.com/58348703/131070862-67b818b2-a2ec-487a-9744-2bf5e0624e65.png)
![7](https://user-images.githubusercontent.com/58348703/131070870-2f1fe2f0-dce4-4a12-adc5-182d4c2f870e.png)
![8](https://user-images.githubusercontent.com/58348703/131070871-f8e2b590-ff4c-41af-b8a7-a12eba835693.png)
![9](https://user-images.githubusercontent.com/58348703/131070872-ad99eebe-62cb-424d-aef6-592909a918d8.png)

**Comment by IngoMeyer441** [comment]
I tried this on my development machine (Manjaro Linux, kitty terminal and tmux 3.2a), but could not reproduce this behavior yet. I am going to install your environment on a fresh macOS VM. Maybe, there are some differences between platforms or terminal emulators.

**Comment by IngoMeyer441** [comment]
I could reproduce the problem on macOS with Alacritty and the default terminal and on Ubuntu 20.04 with the GNOME terminal. So it seems to be indepedent from the operating system and the terminal emulator.

**Comment by Futurile** [comment]
I think it's something to do with the terminal emulator, rather than the OS. 

- XTerm (330) - fails - it jumps to the end of the scrollback [0/12] in the top right
- XTerm (370) - fails - installed from Guix - same as above
- Gnome Terminal (3.28.2) - fails - this is the Ubuntu 18.04 LTS one - same as XTerm
- Alacritty 0.9.0 - fails - difference is it jumps to the search location, but doesn't keep the highlight
- Kitty 0.20.3 - doesn't work - same as XTerm

```
My thoughts meander along (blown round unbound) the poets verse:

        I wandered lonely as a cloud
        That son high o'er vales and hills,
        When all at once I saw a crowd,
        A host, of golden daffodils;
        Beside the lake, beneath the trees,
        Fluttering and dancing in the breeze.

```
* Enter tmux-copy mode: <Prefix>
* Move using the vim keys to some line: start of line 4 'A host
* Start a selection (for me this is v): v
* Do a tmux-jump search for a character, say d: sd
* Fails here if there's more than one pane
* Copy the highlighted text: y

**Comment by IngoMeyer441** [comment]
@Futurile Thanks for your tests and your further investigation! :slightly_smiling_face: 
I tried your instructions on Manjaro Linux, kitty 0.24.1 and tmux 3.2a, but still I cannot reproduce this error on my development machine (but I suspect it also broken for me on my Mac VM).
I really would like to fix this issue, but I have still no idea what is going on here...

**Comment by IngoMeyer441** [comment]
Does tmux-easy-motion often fail for you @butwerenotthereyet and @Futurile when multiple panes are involved?

**Comment by butwerenotthereyet** [comment]
@IngoMeyer441 Yes, it fails very often when there are multiple panes involved.

**Comment by veyzz** [comment]
Gentoo, kitty 0.26.5, tmux 3.3a.

The same issue.
